### PR TITLE
docs: update flink_application_version docs

### PR DESCRIPTION
**Pull request by jeff-held-aiven**
- Fix extraneous quotation marks
- Remove references to deprecated schema values and use current ones instead (`sinks` and `sources` -> `sink` and `source`)

## Why this way
We shouldn't display invalid code in our examples

### CODE REVIEW

**Review by byashimov — DISMISSED** [review]
Hey @jeff-held-aiven Thanks for the PR. 
1. If you don't mind I would merge it later as we have PR with tags that already have been released. We can't rebase them without ruining the hash sums.
2. We haven't discussed this yet, and it might be a good time to rise this question, but imho the changelog is a thing that you're looking at to decide if there is anything important to be aware of before the upgrade. I would not recommend to log documentation changes

Given 1), and truly optional 2) I'll block the PR until everything is merged to prevent issues. I'll comeback shortly when main is ready for this. Thanks.

**Comment by Serpentiel** [comment]
@jeff-held-aiven any news in regards to this PR? If not, I propose to close it.

**Comment by byashimov** [comment]
@jeff-held-aiven please resolve the changelog conflict, and we are ready to go :)

**Review by Serpentiel — APPROVED** [review]
lgtm

**Review by Serpentiel — APPROVED** [review]
lgtm

**Pull request merged**
